playbooks/tensorflow-volcano-kubeflow-integration-test-k8s/kubeflow_tfbenchmarks.yaml

```diff
@@ -0,0 +1,217 @@
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi }
+      - { case: "case4", name: "kf-tfbenchmarks-1", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi }
+      - { case: "case4", name: "kf-tfbenchmarks-2", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi }
+      - { case: "case4", name: "kf-tfbenchmarks-3", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi }
+      - { case: "case4", name: "kf-tfbenchmarks-4", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi }
+      - { case: "case4", name: "kf-tfbenchmarks-5", ps_replicas: 2, worker_replicas: 4,
+          ps_cpu: 1000m, ps_mem: 1000Mi, worker_cpu: 2000m, worker_mem: 3800Mi }
+  tasks:
+    - name: prepare k8s namespaces for kubeflow and TFjobs
+      shell:
+        cmd: |
+          set -ex
+          cat <<EOF | kubectl create -f -
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+           name: kubeflow
+           annotations:
+             scheduler.alpha.kubernetes.io/node-selector: env=system
+          spec: {}
+          status: {}
+          EOF
+
+          cat <<EOF | kubectl create -f -
+          apiVersion: v1
+          kind: Namespace
+          metadata:
+           name: kf-tfbenchmarks
+           annotations:
+             scheduler.alpha.kubernetes.io/node-selector: env=workload
+          spec: {}
+          status: {}
+          EOF
+          CURRENT_CONTEXT=$(kubectl config current-context)
+          CURRENT_CLUSTER=$(kubectl config get-contexts $CURRENT_CONTEXT | tail -1 | awk '{print $3}')
+          CURRENT_USER=$(kubectl config get-contexts $CURRENT_CONTEXT | tail -1 | awk '{print $4}')
+          kubectl config set-context kf-tfbenchmarks \
+            --namespace kf-tfbenchmarks \
+            --cluster $CURRENT_CLUSTER \
+            --user $CURRENT_USER
+          sed -i '/--enable-admission-plugins/ s/$/,PodNodeSelector/' /etc/kubernetes/manifests/kube-apiserver.yaml
+          sleep 30
+        executable: /bin/bash
+
+    - name: config PVs for kubeflow deployment
+      shell:
+        cmd: |
+          set -ex
+          apt-get install -y nfs-server
+          export KUBECONFIG=/etc/kubernetes/admin.conf
+          for i in `seq 1 3`;do
+              mkdir -p /nfs-data/kubeflow-pv$i
+              echo "/nfs-data/kubeflow-pv$i *(rw,sync,no_root_squash,no_subtree_check)" >> /etc/exports
+              systemctl restart nfs-kernel-server.service
+              cat << EOF >> kubeflow-pv$i.yaml
+              apiVersion: v1
+              kind: PersistentVolume
+              metadata:
+                name: kubeflow-pv$i
+              spec:
+                capacity:
+                  storage: 20Gi
```

Comment:
Question: How many G disks this case required? 80G or 100G? OpenLab node label `ubunut-xenial` is ready for this?

Reply:
the kubeflow need 3 PVs, one need 10G and another two need 20G, here I simply create 3 * 20G for 3 PVs, I think it is OK even if there isn't so much disk volume. because the 20G is the max available, in fact, it is unlikely to use so much. (not very sure...)

Reply:
OK, got it